src/html/partials/header.html

```diff
@@ -1,18 +1,25 @@
 <header class="header">
 
-  <div class="wrapper--title">
+  <div class="header-wrapper">
+
+    <div class="header__background-img img-1"></div>
+    <div class="header__background-img img-2"></div>
+    <div class="header__background-img img-3"></div>
+    <div class="header__background-img img-4"></div>
+
     <h1 class="header__title">A Viget Exploration</h1>
-  </div>
 
-  <img class="header__img" src="../../images/TitleTreatment-01.svg" alt="improving medical adherence">
+    <img class="header__img" src="../../images/TitleTreatment-01.svg" alt="improving medical adherence">
 
-  <nav role="navigation" class="header__nav">
-    <ul class="header__nav-list">
-      <li><a href="#problem">The Problem</a></li>
-      <li><a href="#key-factors">Key Factors</a></li>
-      <li><a href="#question">The Question</a></li>
-      <li><a href="#solution">The Solution</a></li>
-    </ul>
-  </nav>
+    <nav role="navigation" class="header__nav">
```

Comment:
no need for `role="navigation"` when using `<nav>`; this is automatically inferred
(https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Navigation_Role)